x-pack/test/upgrade_assistant_integration/upgrade_assistant/reindexing.js

```diff
@@ -13,7 +13,7 @@ import { getIndexState } from '../../../plugins/upgrade_assistant/common/get_ind
 export default function ({ getService }) {
```

Comment:
I didn't realize these tests existed until today 😄 . We also have UA tests in `x-pack/test/api_integration`. In a future PR, it might be worth seeing if we can consolidate.